frontend/package.json

```diff
@@ -36,6 +39,7 @@
     "vue-tsc": "^1.0.24"
   },
   "dependencies": {
+    "@acaldas/document-model-libs": "^1.0.15",
```

Comment:
One thing that I haven't noticed before: this package actually have to be added to the `api` service instead of the frontend (since it will be used there in the first place) – otherwise we again may end up in the same situation where package version which is used by the api is different from the documentation displayed on the frontend

Reply:
It does not make 100% sense to me.

If the dependency is in the api , then the api has to generate and serve the readme file - correct?

This then implies that i would have to write logic that handles this: 
  - provides the readme in some format via the api 
  - gets the readme from the api and displays this on the frontend.

Correct? Or did you have something else in mind?

Reply:
Ok, now the dep is in the api. Docker-compose works for me.